### ISSUE TITLE
Unstable formatting on parenthesized multiline string and method call with args.

### ISSUE THREAD

**Issue by brandtbucher** [psf/black#958]
**Operating system:** Ubuntu 16.04
**Python version:** Python 3.7.4
***Black* version:** e664517
**Does also happen on master:** Yes

I've verified that this is a regression from 19.3b0. This is as much as I can minimize:

```diff
--- source
+++ first pass
@@ -1,3 +1,3 @@
-("""
-""")._(_)
+"""
+"""._(_)
 
--- first pass
+++ second pass
@@ -1,3 +1,5 @@
 """
-"""._(_)
+"""._(
+    _
+)
```

**Comment by ichard26** [comment]
Closing as this is "fixed" on master by commit 14b28c89c22659e1f935bc0ac22ee03d90bcc290 (PR #1079):

```console
(black) ichard26@acer-ubuntu:~/programming/oss/black$ cat temp.py
("""
""")._(_)
(black) ichard26@acer-ubuntu:~/programming/oss/black$ git checkout 14b28c89c22659e1f935bc0ac22ee03d90bcc290
Previous HEAD position was a73d258 fix CI (#1078)
HEAD is now at 14b28c8 Back out #850 (#1079)
(black) ichard26@acer-ubuntu:~/programming/oss/black$ python black.py temp.py --diff
--- temp.py	2020-09-18 22:59:21.156658 +0000
+++ temp.py	2020-09-18 23:07:36.522427 +0000
@@ -1,3 +1,5 @@
-("""
-""")._(_)
+(
+    """
+"""
+)._(_)
 
reformatted temp.py
All done! ✨ 🍰 ✨
1 file reformatted.
(black) ichard26@acer-ubuntu:~/programming/oss/black$ git checkout HEAD~1
Previous HEAD position was 14b28c8 Back out #850 (#1079)
HEAD is now at a73d258 fix CI (#1078)
(black) ichard26@acer-ubuntu:~/programming/oss/black$ python black.py temp.py --diff
error: cannot format temp.py: INTERNAL ERROR: Black produced different code on the second pass of the formatter.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_qp2po0e_.log
Oh no! 💥 💔 💥
1 file failed to reformat.
(black) ichard26@acer-ubuntu:~/programming/oss/black$ cat /tmp/blk_qp2po0e_.log 
--- source
+++ first pass
@@ -1,3 +1,3 @@
-("""
-""")._(_)
+"""
+"""._(_)
 
--- first pass
+++ second pass
@@ -1,3 +1,5 @@
 """
-"""._(_)
+"""._(
+    _
+)
```

I say "fixed" because the output you see on commit 14b28c89c22659e1f935bc0ac22ee03d90bcc290 (and on master too) is  wonky. I don't think this is acceptable:

```diff
-("""
-""")._(_)
+(
+    """
+"""
+)._(_)
```

I'll open a new issue for this if there isn't one already..

**Comment by ichard26** [comment]
> I'll open a new issue for this if there isn't one already..

Took *way too long*, but here it is: GH-2113.
